### DISCUSSION TITLE
WinEdt Help: the section about Execution Modes is possibly misplaced

### DISCUSSION THREAD

**Discussion by tozGitHub** [WinEdt-Team/WinEdt-Support#38] [1.1 General Help and Discussion]
Dear Alex, dear community

WinEdt Configuration Manual|Advanced Configuration...
Is it on purpose that Execution Modes... is missing?
(I see that there is exhaustive material under WinEdt Manual|Execution Modes, but maybe one would also expect at least a link under Advanced Configuration).

Best,
Thomas

**Comment by Alex-Simonic** [comment]
As there is a long Chapter on Execution Modes I see no need for any extra info or link here...
